crates/utils/src/settings/mod.rs

```diff
@@ -23,15 +22,6 @@ pub static SETTINGS: LazyLock<Settings> = LazyLock::new(|| {
   }
 });
 
-#[allow(clippy::expect_used)]
-static WEBFINGER_REGEX: LazyLock<Regex> = LazyLock::new(|| {
-  Regex::new(&format!(
-    "^acct:([a-zA-Z0-9_]{{3,}})@{}$",
-    SETTINGS.hostname
-  ))
-  .expect("compile webfinger regex")
-});
```

Comment:
I grepped but couldn't find a copy of this anywhere. Does that mean this check got removed?

Reply:
It was moved to the apub library.